src/app/modules/ix-table2/components/ix-table-body/cells/ix-cell-date/ix-cell-date.component.html

```diff
@@ -0,0 +1 @@
+{{ formatDate }}
```

Comment:
May be better to use `formatDateTime` pipe. It's not great, but it takes into account date format specified by user in settings.